Documentation/docs/using-fluent.md

```diff
@@ -0,0 +1,220 @@
+        <Border Background="{DynamicResource {x:Static SystemColors.AccentColorLight1BrushKey}}" />
+        <Border Background="{DynamicResource {x:Static SystemColors.AccentColorLight2BrushKey}}" />
+        <Border CornerRadius="0 2 2 0" Background="{DynamicResource {x:Static SystemColors.AccentColorLight3BrushKey}}" />
+    </StackPanel>
+```
+
+> [!NOTE]
+> When using AccentColor APIs directly, use Light1, Light2 and Light3 variations for Dark mode and Dark1, Dark2 and Dark3 for light mode in your application.
+
+## Backdrop support in WPF
+
+Unlike WinUI, WPF does not have `Acrylic` and `Mica` material, controllers or brushes. For now, to enable backdrop in WPF, we took advantage of the Desktop Window Manager ( DWM ) for specifying the system-drawn backdrop material on a window. We do this by calling the [DwmSetWindowAttribute function](https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/nf-dwmapi-dwmsetwindowattribute) for the window.
+
+We haven't provided any API yet, for switching between the system defined backdrop types ( see [DWM_SYSTEMBACKDROP_TYPE](https://learn.microsoft.com/en-us/windows/win32/api/dwmapi/ne-dwmapi-dwm_systembackdrop_type) ), as we were not sure of the path we are going to take for this feature.
+
+Rather, when Fluent theme is enabled we set Mica ( DWMSBT_MAINWINDOW ) backdrop on the Window. However, since we need to modify `PresentationSource.CompositionTarget.Background` and call the `DwmExtendFrameIntoClientArea` method, we have provided an  app-context switch to disable the applicaiton of backdrop on windows : `Switch.System.Windows.Appearance.DisableFluentThemeWindowBackdrop`.
+
+Developers can disable backdrop in their applications by including the switch as following in their project files : 
+```xml
+<ItemGroup>
+    <RuntimeHostConfigurationOption Include="Switch.System.Windows.Appearance.DisableFluentThemeWindowBackdrop" Value="True" />
+</ItemGroup>
+```
+
+or including it in your runtime config file as follows : 
+
+```json
+{
+  "runtimeOptions": {
+    "tfm": "net9.0",
+    "frameworks": [
+        // specifications...   
+    ],
+    "configProperties": {
+      "Switch.System.Windows.Appearance.DisableFluentThemeWindowBackdrop": true,
+    }
+  }
+}
+```
+
+## Going ahead
+
+There are a lot of issues in the styles and the current infrastructure of how Fluent theme is loaded, etc. and we will keep working iteratively to resolve these issues in .NET 10. There are still a lot of features that are still missing that we will on in the next iteration.
+
+Meanwhile, I would like to ask the community to go ahead and try out the new theme in their applications, and provide us with feedbacks via GitHub issues, discussions, feature suggestions and PRs to make it better.
+
+> [!NOTE]
+> While doing so, add the **"Win 11 Theming"** label to your contributions, so that it becomes easier for us to track and prioritize your issues.
```

Comment:
External developers can't set labels, right?

Reply:
My bad, I will update that part.